commerce-product-service/src/main/java/com/liferay/commerce/product/internal/util/data/provider/CPInstanceOptionsValuesDataProvider.java

```diff
@@ -70,9 +69,6 @@ public DDMDataProviderResponse getData(
 			DDMDataProviderRequest ddmDataProviderRequest)
 		throws DDMDataProviderException {
 
-		DDMDataProviderResponse.Builder ddmDataProviderResponseBuilder =
-			DDMDataProviderResponse.Builder.newBuilder();
```

Comment:
@marco-leo I think these changes may break 7.2 compatibility. We should test to verify.